carmajava/launch/environment.launch

```diff
@@ -18,13 +18,132 @@
 	Launch file for launching the nodes in the CARMA environmental perception stack
 -->
 <launch>
+ 
+  <!-- vector map loader -->
+  <group>
   <arg name="vector_map_file" default="vector_map.osm" doc="Path to the map osm file if using the noupdate load type"/>
 
-  <!-- vector map loader -->
   <include file="$(find map_file)/launch/lanelet2_map_loader.launch">
    <arg name="file_name" value="$(arg vector_map_file)"/>
   </include>
+  </group>
+
+<!-- vision_darknet_detect -->
+  <group>
+  <include file="$(find vision_darknet_detect)/launch/vision_yolo3_detect.launch">
+  <arg name="gpu_device_id" default="0"/>
+  <arg name="score_threshold" default="0.30"/>
+  <arg name="nms_threshold" default="0.30"/>
+
+  <arg name="network_definition_file" default="$(find vision_darknet_detect)/darknet/cfg/yolov3.cfg"/>
+  <arg name="pretrained_model_file" default="$(find vision_darknet_detect)/darknet/data/yolov3.weights"/>
+  <arg name="names_file" default="$(find vision_darknet_detect)/darknet/cfg/coco.names"/>
```

Comment:
We need to have the path for these yolo weights set to somewhere on the vehicle that the user can access through a docker volume. We can't have them in our docker files due to licensing issues. 
Lets dicscuss